pkgs/development/libraries/cudd/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "cudd";
+  version = "3.0.0";
+
+  src = fetchurl {
+    url = "https://davidkebo.com/source/cudd_versions/cudd-3.0.0.tar.gz";
+    sha256 = "0sgbgv7ljfr0lwwwrb9wsnav7mw7jmr3k8mygwza15icass6dsdq";
+  };
+
+  configureFlags = [
+    "--enable-dddmp"
+    "--enable-obj"
+  ];
+
+  patches = [
+    ./cudd.patch
```

Comment:
Is this patch submitted upstream? Maybe we can comment link the issue/report?

Reply:
it doesn't currently seem like upstream is maintained. i might take over actual maintainership. this just pulls the last stable version. since it's a data structures library it shouldn't need updating often really.